src/utils/stringToNumber.js

```diff
@@ -0,0 +1,12 @@
+export default function stringToNumber(string) {
+  try {
+    const changedString = Number(string);
+    if (isNaN(changedString)) {
```

Comment:
그리고 isNaN을 사용한다면 이렇게 바꾸는 것이 더 괜찮을까요??
```suggestion

    if (isNaN(string)) {
        const changedString = Number(string);
```